pkg/mediationcontainer/client_websocket_transport.go

```diff
@@ -19,7 +20,8 @@ const (
 	Ready            TransportStatus = "ready"
 	handshakeTimeout                 = 60 * time.Second
 	wsReadLimit                      = 33554432 // 32 MB
-	writeWaitSeconds                 = 120
+	writeWaitTimeout                 = 120 * time.Second
+	pingPeriod                       = 60 * time.Second
```

Comment:
From the comment below, sending the ping is needed to ensure knowing the disconnection, when happening. How do we set the ping period? Is the 60s related to other periods, like connection retry (30s)?

Reply:
no, it is not related to the retry. It is used to trigger the [websocket.ReadMessage](https://github.com/turbonomic/turbo-go-sdk/blob/36a45086d337dd115b0bfd737060c395a6fb3411/pkg/mediationcontainer/client_websocket_transport.go#L161) return when there is an error in the connection.

Without the Ping thread, some times this `websocket.ReadMessage` won't return even if the server has restarted.